doc/source/api/diffpy.nmf_mapping.rst

```diff
@@ -1,28 +1,30 @@
 :tocdepth: -1
 
-diffpy.nmf-mapping package
```

Comment:
You seem to be working on files that show as "modified" in your git status list which we normally leave until the end and do very carefully.  This is ok to do them now, since you started, but in general, type `git status` and work on files that are pure "adds" (i.e., in the skpg package but not the original, which are listed as "untracked" in your list) or pure "keeps" (i.e., in the old package but not the skpkg one, which will show up as "deleted").  As long as you used no-clobber when you did the `cp`, we should be ok. Especially for "modifieds", but if you want to know what is going on in all cases, do a `git diff <filename> to see the changes before you decide what to do.

I think it is ok to go on and continue with this PR as it is, but for future PRs, please pay attention to this.